tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,21 @@
+k = 1
+
+
+def get_digits_number(num: int) -> int:
+    count = 0
+    while num > 0:
+        count += 1
+        num = num // 10
+    return count
+
+
+def is_narcissistic_num(num: int) -> bool:
+    n: int = get_digits_number(num)
+    nth: [int] = [int(digit) ** n for digit in str(num)]
+    return num == sum(nth)
+
+
+while 1 <= k <= 1000:

```

Comment:
Since the range is fixed you could use for loop 
e.g.
`for i in range(1,1001): ` pay attention that the range has an open interval on the right. range(100) = 0<=x<100.

Reply:
reworked